client/termdb/TermdbVocab.js

```diff
@@ -572,11 +584,13 @@ export class TermdbVocab extends Vocab {
 		}
 
 		// use same query method for all dictionary terms
+		const tw = await fillTermWrapper({ term, q: _body.term1_q || {} }, this)
```

Comment:
this seems heavy duty. i would like to know the purpose

Reply:
The purpose is to supply a termwrapper to the `termdb/categories` route. `fillTermWrapper` will also compute a `$id` based on the `term` and `q`, rather than using a random string.

Reply:
please add it as a comment